reports/api/endpoints_test.go

```diff
@@ -832,3 +832,572 @@ type respBody struct {
 	ID      string         `json:"id"`
 	Status  reports.Status `json:"status"`
 }
+
+const (
+	validTemplate = `<!DOCTYPE html>
```

Comment:
Instead of writing more than one template, use a valid one and `strings. Replace` to create invalid templates.